src/Processor/SetProcessor.php

```diff
@@ -25,9 +28,11 @@ public function __construct(string $field, string $value)
     /**
      * Set field value.
      *
+     * @param mixed $value
+     *
      * @return $this
      */
-    public function setValue(string $value): self
+    public function setValue($value): self
```

Comment:
That's would be a BC break for anyone who reimplemented this method and kept the `string $value` signature.
But it should be very limited as extending this processor is probably not widespread at all.

Reply:
Agree, but I also hope this is very rare. I suggest we mark it as a breaking change in the changelog with a note on when it would be breaking (someone inheriting SetProcessor).

It is one of these scenarios I think where in theory it is a breaking change and I'm pretty sure someone is going to bring up that because of that we (I) don't follow semver but working around it would not bring benefits to the large majority of users. I'm +1 on the change.

Reply:
I'll update the PR later adding it in the BC Break section, it wasn't intentional, I changed it in the constructor which is not BC break and then changed it in the `setValue` without thinking about that.